Description: Fix unicode strings in pbr/tests/test_setup.py
 Avoids failure in Python 3.2.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-04-11

--- a/pbr/tests/test_setup.py
+++ b/pbr/tests/test_setup.py
@@ -20,6 +20,7 @@
 import sys
 import tempfile
 import testscenarios
+import six
 
 try:
     import cStringIO as io
@@ -163,10 +164,10 @@
             self.assertNotIn('1_foo.1', changelog_contents)
 
     def test_generate_authors(self):
-        author_old = u"Foo Foo <email@foo.com>"
-        author_new = u"Bar Bar <email@bar.com>"
-        co_author = u"Foo Bar <foo@bar.com>"
-        co_author_by = u"Co-authored-by: " + co_author
+        author_old = six.u("Foo Foo <email@foo.com>")
+        author_new = six.u("Bar Bar <email@bar.com>")
+        co_author = six.u("Foo Bar <foo@bar.com>")
+        co_author_by = six.u("Co-authored-by: ") + co_author
 
         git_log_cmd = (
             "git --git-dir=%s log --format=%%aN <%%aE>"
